yaml/schema/mod.ts

```diff
@@ -4,8 +4,32 @@
 // Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
 // This module is browser compatible.
 
+import { CORE_SCHEMA } from "./core.ts";
 export { CORE_SCHEMA } from "./core.ts";
+import { DEFAULT_SCHEMA } from "./default.ts";
 export { DEFAULT_SCHEMA } from "./default.ts";
+import { EXTENDED_SCHEMA } from "./extended.ts";
 export { EXTENDED_SCHEMA } from "./extended.ts";
+import { FAILSAFE_SCHEMA } from "./failsafe.ts";
 export { FAILSAFE_SCHEMA } from "./failsafe.ts";
+import { JSON_SCHEMA } from "./json.ts";
 export { JSON_SCHEMA } from "./json.ts";
+
+export function replaceSchemaNameWithSchemaClass(
```

Comment:
Perhaps more of a matter of preference, I would've preferred this to be `getSchema(schema: string | Schema): Schema` and use `switch (schema)`.